src/backends/torch/torchlib.cc

```diff
@@ -126,7 +128,9 @@ namespace dd
 
   c10::IValue TorchModule::forward(std::vector<c10::IValue> source)
   {
-    if (_native) // native modules take only one tensor as input for now
+    if (_graph) // native modules take only one tensor as input for now
+      return _graph->forward(to_tensor_safe(source[0]));
+    if (_native)
```

Comment:
Can we have both, or do we want an `else` here ?

Reply:
I followed what was previously done with traced net that can followed by a weight net (fully connected) ; This allows for free to chain nets, if you really know what you are doing (typically few things will need to be rewritten around input connector -> first net) 
For know we do not have both, and at some time in the future, this should be superseded by the graph abstraction layer, ie a torchModule will be a node , that can be of one type native / traced / or (sub) graph.
Morality : if you think having a else makes things more clear, I am ok ;)